cmd/guacone/cmd/files.go

```diff
@@ -190,3 +196,23 @@ func getAssembler(opts options) (func([]assembler.Graph) error, error) {
 		return nil
 	}, nil
 }
+
+func createIndices(client graphdb.Client) error {
+	indices := map[string][]string{
+		"Artifact":    {"digest", "name"},
+		"Package":     {"purl", "name"},
+		"Metadata":    {"id"},
+		"Attestation": {"digest"},
+	}
```

Comment:
nit: thinking if this should be part of the assmbler

Reply:
Most likely will move there in the future. I'm currently looking at usages of `GuacNode` and `GuacEdge` throughout the codebase to try to see which places cause duplication and don't have good UX (e.g., there's a very large duplication in `internal/testing/ingestor/testdata`, as a very stringent example, albeit in testing). I'm planning some refactoring on these after the POC.